/*
 *  Copyright (c) 2016 Leo Khotov
 *
 *  This program is free software: you can redistribute it and/or modify
 *  it under the terms of the GNU Lesser General Public License as published by
 *  the Free Software Foundation, either version 3 of the License, or
 *  (at your option) any later version.
 *
 *  This program is distributed in the hope that it will be useful,
 *  but WITHOUT ANY WARRANTY; without even the implied warranty of
 *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 *  GNU Lesser General Public License for more details.
 *
 *  You should have received a copy of the GNU Lesser General Public License
 *  version 3 along with this work.  If not, see <http://www.gnu.org/licenses/>.
 *
 */
syntax = "proto3";

package roosha;

option java_package = "com.github.roosha.proto";
option java_outer_classname = "CommonsProto";

option cc_enable_arenas = false;

option optimize_for = SPEED;

message Void{}

message Credentials {
	string login = 1;
	string passwordHash = 2;	
}

message AuthenticationToken {
	string token = 1;
}
